packages/frontend/src/components/common/Tree/TreeItem.tsx

```diff
@@ -109,7 +104,7 @@ const TreeItem: React.FC<Props> = ({
                 {stringLabel}
             </Highlight>
 
-            {!isRoot && selected && (
+            {selected && (
```

Comment:
<img width="581" alt="image" src="https://github.com/user-attachments/assets/4dd99a14-1bcd-4c52-900e-2d32cf39c48a" />

Should we highlight the row as well when 'All spaces' is selected?